### PR TITLE
NMS-13758: Split Database Reports docs into multiple pages

### PR DESCRIPTION
**Pull request by mfuhrmann**
### All Contributors

* [x] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [x] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### Contribution Checklist

* Please [make an issue in the OpenNMS issue tracker](https://issues.opennms.org) if there isn't one already.<br />Once there is an issue, please:
  1. update the title of this PR to be in the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
  4. once you've created this PR, please link to it in a comment in the JIRA issue
  Don't worry if this sounds like a lot, we can help you get things set up properly.
* If this is a new or updated feature, is there documentation for the new behavior?
* If this is new code, are there unit and/or integration tests?
* If this PR targets a `foundation-*` branch, does it try to avoid changing files in `$OPENNMS_HOME/etc/`?

### What's Next?

A PR should be assigned at least 2 reviewers.  If you know that someone would be a good person to review your code, feel free to add them.

If you need help making additions or changes to the documentation related to your changes, please let us know.

In any case, if anything is unclear or you want help getting your PR ready for merge, please don't hesitate to say something in the comments here,
or in [the #opennms-development chat channel](https://chat.opennms.com/opennms/channels/opennms-development).

Once reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the OGP) you are welcome to merge the PR when you're ready.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-13758

### CODE REVIEW

**Comment by mfuhrmann** [comment]
I had a talk with @indigo423 about this document. Most of the stuff is outdated and he is pretty sure it does not work as documented. For now, it doesn't make sense to spend too much time on it. I've followed @indigo423 advice to just split it like this for now.

**Review by Bonrob2 — APPROVED** [review]
Approval retracted, I realized why the nav seemed off, so have updated the heading levels. Please verify they are rendering correctly in the Table of Contents. Each file has to start with a single "=" in order to build properly.

File: docs/modules/operation/pages/database-reports/pdf-reports-grafana.adoc [hunk 1]
```diff
@@ -0,0 +1,70 @@
+== Creating PDF reports from Grafana dashboards using {page-component-title}
```
Comment (Bonrob2) [inline docs/modules/operation/pages/database-reports/pdf-reports-grafana.adoc:1]:
```suggestion
= Creating PDF reports from Grafana dashboards using {page-component-title}
```

File: docs/modules/operation/pages/database-reports/pdf-reports-grafana.adoc [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+* Keep staff without access to {page-component-title} informed about network performance for improved capacity planning
+* Create a permanent record of strategic information and progress over a long period of time
+
+The PDF report displays each of the panels from the specified dashboard, with one, two, or four panels per page, depending on the selected template.
+
+Dashboard to PDF:
+
+image::database-reports/Dashboard-pdf.png[PDF report]
+
+=== Before you begin
```
Comment (Bonrob2) [inline docs/modules/operation/pages/database-reports/pdf-reports-grafana.adoc:15]:
```suggestion
== Before you begin
```

File: docs/modules/operation/pages/database-reports/pdf-reports-grafana.adoc [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+This feature requires {page-component-title} and an instance of Grafana with at least one dashboard and panel.
+OpenNMS lets you create a report for any Grafana dashboard, not just those created using link:http://docs.opennms.org/helm/branches/master/helm/latest/welcome/index.html[OpenNMS Helm].
+
+You must set up Grafana as a data source by xref:grafana-endpoint[configuring the Grafana endpoint] in {page-component-title}.
+
+[[grafana-endpoint]]
+=== Configure the Grafana endpoint
```
Comment (Bonrob2) [inline docs/modules/operation/pages/database-reports/pdf-reports-grafana.adoc:23]:
```suggestion
== Configure the Grafana endpoint
```

File: docs/modules/operation/pages/database-reports/pdf-reports-grafana.adoc [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
++
+image::database-reports/grafana-endpoint.png[Endpoint, 400]
+
+. In the *Endpoint Configuration* screen click the plus sign on the right to add a new endpoint.
+. Fill in the information and click *Test Connection*.
+. Click *Create*.
+
+You can now use {page-component-title} to create PDF reports of Grafana dashboards.
+
+=== Create a PDF of a Grafana dashboard
```
Comment (Bonrob2) [inline docs/modules/operation/pages/database-reports/pdf-reports-grafana.adoc:47]:
```suggestion
== Create a PDF of a Grafana dashboard
```

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
Headings were the wrong level.

**Comment by mfuhrmann** [comment]
If green, feel free to merge.

**Review by Bonrob2 — APPROVED** [review]

**Pull request merged**